qdrant_client/local/local_collection.py

```diff
@@ -213,7 +226,11 @@ def search(
 
         vectors = self.vectors[name]
         params = self.get_vector_params(name)
-        scores = calculate_distance(vector, vectors[: len(self.payload)], params.distance)
+        if isinstance(query_vector, np.ndarray):
+            scores = calculate_distance(query_vector, vectors[: len(self.payload)], params.distance)
+        else:
+            scores = calculate_best_scores(query_vector, vectors[: len(self.payload)], params.distance) 
```

Comment:
i'd rather do the condition explicitly `if isinstance(query_vector, np.ndarray) + elif isinstance(calculate_best_scores, RecoQuery) + else raise`

despite it is already handled in  `name, query_vector = self._resolve_query_vector_name(query_vector)`, i think it just makes the code more readable